src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -61,7 +61,7 @@ To learn more, see [User management concepts](/docs/accounts/accounts-billing/ne
 
 If you'll need to do a good amount of partitioning of users' access to accounts or roles, you'll want to think about how you'll implement groups for your organization. Here are some tips for how different types of organizations may use groups:  
 
-* For organizations with a good number of accounts, a common configuration is to have one user group with the [**Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles) (organization-level management capabilities), one group with the [**Authentication domain manager**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles) (user management capabilities), a group with the [**Billing user**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles) (billing-related capabilities) on the primary (first) account, and then configuring various users groups for admins, users, and other roles as needed on your other accounts.  
+* For organizations with a good number of accounts, a common configuration is to have one user group with the [**Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles) (organization-level management capabilities), one group with the [**Authentication domain manager**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles) (user management capabilities), a group with the [**Billing user**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles) (billing-related capabilities) on the primary (first) account, and then configuring various users groups for admins, users, and other roles as needed on your other accounts.  
```

Comment:
@zuluecho9, the billing user link didn't work for me locally.

Reply:
they're working. not sure what's going on. local builds are really a problem.